src/orquestra/quantum/api/wavefunction_simulator_contracts.py

```diff
@@ -0,0 +1,228 @@
+################################################################################
+# © Copyright 2022 Zapata Computing Inc.
+################################################################################
+from functools import partial
+from itertools import chain
+
+# Test values of expectation values
+# Number of circuits run increases
+# Number of jobs_executed increases
+import numpy as np
+
+from ..circuits import CNOT, Circuit, H, X, builtin_gate_by_name
+from ..operators import PauliTerm
+from ..testing.test_cases_for_backend_tests import (
+    one_qubit_non_parametric_gates_amplitudes_test_set,
+    one_qubit_parametric_gates_amplitudes_test_set,
+    two_qubit_non_parametric_gates_amplitudes_test_set,
+    two_qubit_parametric_gates_amplitudes_test_set,
+)
+from .wavefunction_simulator import WavefunctionSimulator
+
+# Length of wavefunction matches 2 ** number_of_quits
```

Comment:
[question] Do you think these comments are necessary?

Reply:
They are leftovers from the drafting process. I removed them in [8d9349a](https://github.com/zapatacomputing/orquestra-quantum/pull/66/commits/8d9349ae5d02f624484837a822a7ff80aa2914ef)